lib/ios/AirMaps/AIRMap.m

```diff
@@ -62,6 +63,7 @@ @implementation AIRMap
 
 - (instancetype)init
 {
+    NSLog(@"AirMap.init................");
```

Comment:
Can we remove all `NSLog` instances ?

Reply:
Sure, this one somehow slipped my attention..

Reply:
@zavadpe  there are more :)

Reply:
Yeah, just found them :)